memory/src/main/java/org/opensearch/ml/memory/action/conversation/UpdateInteractionTransportAction.java

```diff
@@ -69,7 +69,7 @@ protected void doExecute(Task task, ActionRequest request, ActionListener<Update
 
             cmHandler.updateInteraction(interactionId, updateContent, getUpdateResponseListener(interactionId, listener, context));
         } catch (Exception e) {
-            log.error("Failed to update Interaction " + interactionId, e);
+            log.error("Failed to update Message " + interactionId, e);
```

Comment:
with ID --> let's keep the message pattern consistent.

Reply:
added them